<?xml version="1.0" encoding="utf-8"?>

<launch>
<!--param name="use_sim_time" value="true"/-->
<arg name="file_path" default="$(find osmmap)/maps/"/>
<arg name="file_name" default="simulation.osm"/>
<arg name="origin_lat" default="39.9558622"/>
<arg name="origin_lon" default="116.3105016"/>
<!-- 58.45015716 -->
<arg name="origin_ele" default="54.45015716"/>
<!-- optional : GNSS, PCD, SIM-->
<arg name="navagation_mode" default="SIM"/>

<node pkg="osmmap" type="navagation_node" name="navagation_node" output="screen" >
  <param name="file_path" value="$(arg file_path)"/>
  <param name="file_name" value="$(arg file_name)"/>
  <param name="origin_lat" value="$(arg origin_lat)"/>
  <param name="origin_lon" value="$(arg origin_lon)"/>
  <param name="origin_ele" value="$(arg origin_ele)"/>
  <param name="navagation_mode" value="$(arg navagation_mode)"/>
</node>

<node pkg="rviz" type="rviz" name="rviz" args="-d $(find osmmap)/config/osm.rviz"/>

<node pkg="graph_tool" type="pcdviewer" name="pcdviewer" args="$(find osmmap)/maps/borregasave.pcd"/>

<include file="$(find prius_description)/launch/demo.launch"/>

<node pkg="tf" type="static_transform_publisher" name="baselink_to_rslidar" args="-1.5 0 -2 0 0 0 rslidar base_link 100" />

<!--node pkg="rosbag" type="play" name="rosbag" args=" - -clock -r 2 /home/wangpeng/AI/2022-04-14-15-48-26.bag"/-->

</launch>
